websites/C/CodinGame/presence.ts

```diff
@@ -0,0 +1,199 @@
+			presenceData.state = `${
+				document.querySelector("h1[class='title']").textContent
+			} (${document.querySelector("div[class='chart-inner']").textContent}%)`;
+		} else presenceData.details = "Looking at Practice";
+	} else if (path.includes("/multiplayer")) {
+		if (path.includes("/multiplayer/clashofcode")) {
+			presenceData.details = "Looking at Clash of Code";
+			presenceData.state = `Rank : ${
+				document.querySelector("span[class='rank-value']").textContent
+			} th`;
+		} else if (path.includes("/multiplayer/bot-programming")) {
+			presenceData.details = "Training: Bot Programming";
+
+			try {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} - Rank : ${
+					document.querySelectorAll(
+						"div[id='content-details-league'] > div > div"
+					)[1].textContent
+				}`;
+			} catch {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} - Not yet solved`;
+			}
+		} else if (path.includes("/multiplayer/codegolf")) {
+			presenceData.details = "Training: Code Golf";
+			try {
+				if (
+					document.querySelector("div[class='chart-inner']").textContent !== "0"
+				) {
+					presenceData.state = `${
+						document.querySelector("h1[class='title']").textContent
+					} (${document.querySelector("span[class='rank-value']").textContent}${
+						document.querySelector("span[class='rank-suffix']").textContent
+					}/${document.querySelector("span[class='rank-total']").textContent})`;
+				} else presenceData.state = "Not yet solved";
+			} catch {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} (${document.querySelector("span[class='rank-value']").textContent}${
+					document.querySelector("span[class='rank-suffix']").textContent
+				}/${document.querySelector("span[class='rank-total']").textContent})`;
+			}
+		} else if (path.includes("/multiplayer/optimization")) {
+			presenceData.details = "Training: Optimization";
+			try {
+				if (
+					document.querySelector("div[class='chart-inner']").textContent !== "0"
+				) {
+					presenceData.state = `${
+						document.querySelector("h1[class='title']").textContent
+					} (${document.querySelector("span[class='rank-value']").textContent}${
+						document.querySelector("span[class='rank-suffix']").textContent
+					}/${document.querySelector("span[class='rank-total']").textContent})`;
+				} else presenceData.state = "Not yet solved";
+			} catch {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} (${document.querySelector("span[class='rank-value']").textContent}${
+					document.querySelector("span[class='rank-suffix']").textContent
+				}/${document.querySelector("span[class='rank-total']").textContent})`;
+			}
+		} else presenceData.details = "Looking at Compete";
+	} else if (path.includes("/cooperate"))
+		presenceData.details = "Looking at Cooperate";
```

Comment:
Ditto

Reply:
This still sounds weird

Reply:
This has not been resolved.https://github.com/PreMiD/Presences/pull/6417/files#r915031372